addons/stock/views/stock_package_type_view.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<record model="ir.ui.view" id="stock_package_type_form">
+    <field name="name">stock.package.type.form</field>
+    <field name="model">stock.package.type</field>
+    <field name="arch" type="xml">
+        <form string="Package Type">
+            <sheet>
+                <label for="name"/>
+                <h1>
+                    <field name="name"/>
+                </h1>
+                <group name="delivery">
+                    <group>
+                        <label for="length_uom_name" string="Size"/>
+                        <div class="o_row" name="size">
+                            <field name="packaging_length" placeholder="Length"/>
+                            <span>&#215;</span>
+                            <field name="width" placeholder="Width"/>
+                            <span>&#215;</span>
+                            <field name="height" placeholder="Height"/>
+                            <span><field name="length_uom_name" help="Size: Length &#215; Width &#215; Height"/></span>
+                        </div>
+                        <label for="max_weight"/>
+                        <div class="o_row" name="max_weight">
+                            <field name="max_weight"/>
+                            <span><field name="weight_uom_name"/></span>
+                        </div>
+                        <field name="barcode"/>
+                    </group>
+                </group>
+            </sheet>
+        </form>
+    </field>
+</record>
+
+<record model="ir.ui.view" id="stock_package_type_tree">
+    <field name="name">stock.package.type.tree</field>
+    <field name="model">stock.package.type</field>
+    <field name="arch" type="xml">
+        <tree string="Package Types">
+            <field name="sequence" widget="handle"/>
+            <field name="name"/>
+            <field name="height"/>
+            <field name="width"/>
+            <field name="packaging_length"/>
+            <field name="max_weight"/>
+            <field name="barcode" invisible="1"/>
```

Comment:
useless (or you meant `optional=hide` ? )

Reply:
@Whenrow oops, I didn't notice it. it's from the original code. But I guess you are right, it should be `optional`